docs/faq.asciidoc

```diff
@@ -44,6 +44,26 @@ It also respects other agents attached to your application at the same time.
 === Do I need to re-compile my application?
 No
 
+[float]
+[[recommended-sampling-rate]]
+=== What is the recommended sampling rate?
+There is no one-size-fits-all answer to an ideal sampling rate.
+Sampling comes down to your preferences and your application.
+The more you want to sample, the more network bandwidth and disk space you'll need.
+
+It's important to note that the latency of an application won't be affected much by the agent (in the order of single-digit microseconds),
+even if you sample at 100%.
+However, the background reporter thread has some work to do for serializing and gzipping events.
+If your application is not CPU bound, this shouldn't matter much.
+
+Sample rate can be changed by altering the <<config-transaction-sample-rate>> configuration. 
```

Comment:
I don't think `<<config-transaction-sample-rate>>` can't work. You are trying to refer to another document, so maybe something like:
```
Sample rate can be changed by altering the <<config-core#config-transaction-sample-rate,`transaction_sample_rate` configuration option>>.
```
to try and link to the exact anchor, and if that doesn't work maybe:
```
Sample rate can be changed by altering the `transaction_sample_rate` <<config-core,configuration option>>.
```
In order to try these out, I assume you need to build locally and see how it works, but I am almost certain that the current way will result in a broken link.

Reply:
It actually does work :)

Asciidoc automatically builds the correct link (`../config-core.html#config-transaction-sample-rate`) from the `<<config-transaction-sample-rate>>`

Here's what it looks like:
<img width="673" alt="screen shot 2018-12-04 at 7 05 48 am" src="https://user-images.githubusercontent.com/5618806/49451163-6f0a4f80-f793-11e8-922f-55e8fdccc803.png">

Reply:
Ohh, it resolves directly from an external anchor, nice!!
Well done then, sorry to doubt, just didn't want to get this nasty broken link notification in dev channel :)

Reply:
No worries at all! I appreciate you digging into everything!